lib/licensee/license.rb

```diff
@@ -157,7 +150,14 @@ def title_regex
         Regexp.new string, 'i'
       end
 
-      parts = [title_regex, key]
+      key_regex = begin
+        string = key.sub('-', '[- ]')
+        string.sub!('.', '\.')
```

Comment:
Could potentially use `Regexp.escape` here instead.

Reply:
I might be missing something, but that seems more complicated to me, as some of the escaping Regexp.escape does has to be undone whether it is done before or after the line currently above.